debian/po/pt_BR.po

```diff
@@ -351,6 +351,13 @@ msgid "Finished upgrade of Ubuntu Pro service configuration"
 msgstr "Atualização das configurações de serviço do Ubuntu Pro concluída"
 
 #: ../../uaclient/messages/__init__.py:85
+#, python-brace-format
+msgid ""
+"Detaching Ubuntu Pro. Previously attached subscription was only valid for "
+"Ubuntu {release} ({series_codename}) release."
+msgstr ""
```

Comment:
Some new strings are added to PO files, do we want to also ship pt_BR translations for that? Not sure about the Pro team process, but I'd expect to have all new additions translated during the release process.

Reply:
Our policy is to be pretty relaxed about adding new strings and deleting existing strings.
We should put in a best effort for updating translations for strings that were already translated but got changed, but not block on it.